src/ert/scheduler/lsf_driver.py

```diff
@@ -197,9 +231,7 @@ async def submit(
             script_path = Path(script_handle.name)
         assert script_path is not None
         script_path.chmod(script_path.stat().st_mode | stat.S_IEXEC)
-        arg_resource_requirement = (
-            ["-R", self._resource_requirement] if self._resource_requirement else []
-        )
+        arg_resource_requirement = self._get_resource_requirement_arg()
```

Comment:
I think you can inline this into the making of `bsub_with_args`, then you don't need the temporary variable arg_resource_requirent.